prog/vnet/update_firewall_rules.rb

```diff
@@ -79,12 +65,25 @@ class Prog::Vnet::UpdateFirewallRules < Prog::Base
 
   chain forward_ingress {
     type filter hook forward priority filter; policy drop;
+
+    # Offload to ubi_flowtable. This is used to offload already filtered
```

Comment:
side note, not for now: I wonder if ipsec can be flow-tabled, to any positive effect.